java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -1769,16 +1769,16 @@ public boolean contains(Scalar needle) {
   }
 
   /**
-   * Returns a new ColumnVector of {@link DType#BOOL8} elements containing true if the corresponding
-   * entry in haystack is contained in needles and false if it is not. The caller will be responsible
-   * for the lifecycle of the new vector.
+   * Returns a new ColumnVector of {@link DType#BOOL8} elements containing true if the 
+   * corresponding entry in needles is contained in this column and false if it is not.
+   * The caller is responsible for the lifecycle of the new vector.
    *
-   * example:
+   * Example:
    *
-   *   haystack = { 10, 20, 30, 40, 50 }
-   *   needles  = { 20, 40, 60, 80 }
+   *   col     = { 10, 20, 30, 40, 50 }
+   *   needles = { 20, 40, 60, 80 }
    *
-   *   result = { false, true, false, true, false }
+   *   result = { true, true, false, false }
```

Comment:
This is very confusing to me.  The original intent of this function was to answer the question, "For this haystack of values, tell me row-by-row if _any_ of the needles are found at this row".  In other words, it builds a filter column that will tell you which rows of the original haystack are needles and which are not.  It's a natural extension of the query "find this single needle in this haystack" which tells us row-by-row whether the needle is found.  We're not changing the haystack here, we're just allowing more than one needle to be used in the search.

By having this return a column vector that is **not** the same length as the original vector, that's very odd and surprising to me.  The RAPIDS Accelerator makes calls to this function expecting the original behavior and will be broken by this change.

Reply:
No, the intent of this function is to answer "for this needles values, tell me if each of these values exists in the haystack". So if you have `haystack = [ 1, 2, 3, 4, 5 ]` and `needles = [ 1, 10 ]`, the answer should be `[ true, false ]`. The current API treats `needles` and `haystack` in the opposite way, which is incorrect.

Reply:
This is an odd way to scale the existing function which takes a single needle and searches this haystack of values showing which rows correspond to the needle.  It's not "for this needle search this haystack," it is "for this haystack, search for this needle."  IMO the obvious way to scale that to search for more needles is to keep the haystack as-is and allow more than one needle to be searched per row.

As I mentioned before, this _will_ break the RAPIDS Accelerator if these are swapped.  The failing Java tests show that the proposed change is not how this API was designed to function.

Reply:
I just checked with @jlowe. In the Java side, this API is a member function thus we want to have the result column having the same size as the called column. As such, in Java, this API is called like `valuesToSearch.contains(searchSpace)`. We can keep the same Java API by just switching parameter order in the C++ JNI layer.

IMO, Java API should be renamed as `containedIn`, so we will have `valuesToSearch.containedIn(searchSpace)`, or `needles.containedIn(haystack)`. But this will be another breaking change thus I just propose it but not do it here.